mod_reforged/hooks/skills/racial/alp_racial.nut

```diff
@@ -1,4 +1,17 @@
 ::mods_hookExactClass("skills/racial/alp_racial", function(o) {
+	o.onAdded = function()
+	{
+		local base = this.m.BaseProperties;
```

Comment:
`this.getContainer().getActor().getBaseProperties();`
You'll need to fix this in the other skill files too.